splinter/driver/djangoclient.py

```diff
@@ -141,17 +141,17 @@ def htmltree(self):
         try:
             return self._html
         except AttributeError:
-            self._html = lxml.html.fromstring(self.html.decode('utf-8'))
+            self._html = lxml.html.fromstring(self.html)
             return self._html
 
     @property
     def title(self):
         html = self.htmltree
-        return html.xpath('//title')[0].text_content().strip().encode('utf-8')
+        return html.xpath('//title')[0].text_content().strip()
 
     @property
     def html(self):
-        return self._response.content
+        return self._response.content.decode(self._response.charset)
```

Comment:
No problem, I just wanted to document it so that other 1.7 users know that [0.7.2 fails](https://github.com/cobrateam/splinter/blob/0.7.2/splinter/driver/djangoclient.py#L154). I don't think it's necesary to change anything as it seems that it's already [fixed in master](https://github.com/cobrateam/splinter/blob/master/splinter/driver/djangoclient.py#L172):

    return self._response.content.decode(self._response._charset or 'utf-8')

This should be working in 0.7.3. :smile: